tapir/coop/models.py

```diff
@@ -719,3 +718,63 @@ def populate(
             old_frozen=old_frozen,
             new_frozen=new_frozen,
         )
+
+class ResignedMembership(models.Model):
+    share_owner = models.ForeignKey(
+        ShareOwner, on_delete=models.deletion.CASCADE, verbose_name=_("Shareowner")
+    )
+    cancellation_date = models.DateField(
+        default=timezone.now(),
```

Comment:
Sorry, found another issue. This should be probably with:
```suggestion
        auto_now_add=True,
```
Am I right? [Here](https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.DateField.auto_now_add) is the documentation about it.

This is visible as a warning during application start:
```
web-1          | WARNINGS:
web-1          | coop.ResignedMembership.cancellation_date: (fields.W161) Fixed default value provided.
web-1          |        HINT: It seems you set a fixed date / time / datetime value as default for this field. This may not be what you want. If you want to have the current date as default, use `django.utils.timezone.now`
```